src/seq/fasta.jl

```diff
@@ -38,482 +38,560 @@ typealias FASTARNASeqRecord       RNASeqRecord{FASTAMetadata}
 typealias FASTAAminoAcidSeqRecord AminoAcidSeqRecord{FASTAMetadata}
 
 
-function show{S}(io::IO, seqrec::SeqRecord{S, FASTAMetadata})
-    write(io, ">", seqrec.name, " ", seqrec.metadata.description, "\n")
-    show(io, seqrec.seq)
+function show{	S}(io::IO, seqrec::SeqRecord{S, FASTAMetadata})
+write(io, ">", seqrec.name, " ", seqrec.metadata.description, "\n")
+show(io, seqrec.seq)
 end
 
 
-"Writes a FASTASeqRecord to an IO-stream (and obeys FASTAs max character constraint)"
-function Base.write{T}(io::IO, seqrec::SeqRecord{T, FASTAMetadata})
-    write(io, ">", seqrec.name)
-    if !isempty(seqrec.metadata.description)
-        write(io, " ", seqrec.metadata.description)
-    end
-    write(io, "\n")
-    maxchars = 79
-    counter = 1
-    len = length(seqrec.seq)
-    for nt in seqrec.seq
-        show(io, nt)
-        if counter % maxchars == 0 && counter < len
-            write(io, "\n")
-        end
-        counter += 1
-    end
-    write(io, "\n")
+	"Writes a FASTASeqRecord to an IO-stream (and obeys FASTAs max character constraint)"
+	function 	Base.write{T}(io::IO, seqrec::SeqRecord{T, FASTAMetadata})
+	write(io, ">", seqrec.name)
+	if !isempty(seqrec.metadata.description)
+		write(io, " ", seqrec.metadata.description)
+	end
+	write(io, "\n")
+	maxchars = 79
+	counter = 1
+	len = length(seqrec.seq)
+	for nt in seqrec.seq
+	show(io, nt)
+	if counter % maxchars == 0 && counter < len
+		write(io, "\n")
+	end
+	counter += 1
+end
+write(io, "\n")
 end
 
 
-const _fastaparser_start  = convert(Int , 6)
-const _fastaparser_first_final  = convert(Int , 6)
-const _fastaparser_error  = convert(Int , 0)
-const _fastaparser_en_main  = convert(Int , 6)
+_fastaparser_start  = 6
+_fastaparser_first_final  = 6
+_fastaparser_error  = 0
+_fastaparser_en_main  = 6
+__fastaparser_nfa_targs = Int8[ 0, 0 ,  ]
+__fastaparser_nfa_offsets = Int8[ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ,  ]
+__fastaparser_nfa_push_actions = Int8[ 0, 0 ,  ]
+__fastaparser_nfa_pop_trans = Int8[ 0, 0 ,  ]
```

Comment:
They should be const, though there isn't a big performance impact. Fixed with https://github.com/BioJulia/ragel7/commit/ba684f9ecb0bcb5c318d640ee7e5f34cdd8ce767